x-pack/plugins/triggers_actions_ui/public/application/sections/actions_connectors_list/components/actions_connectors_list.tsx

```diff
@@ -525,7 +525,7 @@ const ActionsConnectorsList: React.FunctionComponent = () => {
             setEditConnectorProps(omit(editConnectorProps, 'initialConnector'));
           }}
           onConnectorUpdated={(connector) => {
-            setEditConnectorProps({ initialConnector: connector });
+            setEditConnectorProps({ ...editConnectorProps, initialConnector: connector });
```

Comment:
setting only the `initialConnector` modifies the editConnectorProps object and causes flickering on the editConnectorFlyout.